REFERENCE.md

```diff
@@ -1,98 +1,107 @@
 # Reference
+
 <!-- DO NOT EDIT: This document was generated by Puppet Strings -->
 
 ## Table of Contents
 
-**Classes**
+### Classes
 
-* [`profile_chrony`](#profile_chrony): Configures Chrony as an NTP Client or Server
-* [`profile_chrony::client`](#profile_chronyclient): Configures chronyd as an NTP client
-* [`profile_chrony::server`](#profile_chronyserver): Configures chronyd as an NTP server
+* [`profile_timesync`](#profile_timesync): Configures Chrony as an NTP Client or Server
+* [`profile_timesync::client`](#profile_timesyncclient): Configures chronyd as an NTP client
+* [`profile_timesync::server`](#profile_timesyncserver): Configures chronyd as an NTP server
```

Comment:
These subclasses no longer exist. Do you need to regenerate the REFERENCE.md file?